tests/main/cgroup-devices/task.yaml

```diff
@@ -0,0 +1,8 @@
+summary: measuring basic properties of device cgroup
+# Ubuntu 14.04 uses a custom build of an older systemd that does not
+# seem to use device cgroups for services. The test is invalid there.
+systems: [-ubuntu-14.04-*]
+execute: ./task.sh
+restore: |
+    rm -f test-snapd-service_1.0_all.snap
```

Comment:
The generic restore for each task makes a clean up on the test dir and removes all the installed snaps. I thinks these lines are not needed.

Reply:
Then they don't hurt and I can easily re-run this in debug shell. Eventually I'd like to have tests clean up after themselves and not rely on the magic global restore to fix things. It is more costly and doesn't really as reliably as test writers assume (see my work on removing snaps on core tests where they otherwise linger).